### PR TITLE
Address feedback on the intro

### LINKED ISSUE #219: Review 05: Chris Wood: Ideas for intro
>The overview of Prio3 and Poplar1 in the introduction feels a bit too out of place, especially as it comes before the generic overview. I would suggest removing this content, as it's somewhat redundant with the section in {{prio3}} and {{poplar1}}.
>
> Moreover, to align with the structure of the document, I might try to reframe the intro to first introduce DAFs, and then introduce malicious clients and the need for verifiability, and then introduce VDAFs. I can propose some text for this if you'd be interested.

### LINKED ISSUE #346: Consider removing definition of DAFs
DAFs are the same as VDAFs except they don't provide robustness on their own. This feature was requested early on in the history of PPM (https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/45). However no DAF has been defined, much less deployed.

We should consider removing DAFs from the draft:
1. This would reduce some maintenance cost in the draft and reference implementation.
2. This would simplify https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/259, which conflicts with the current goal of writing down DAF as a "warm-up" to VDAFs.

### PR DESCRIPTION
**Pull request by cjpatton**
Closes #219.
Closes #346 (the new intro rework leans into DAF being a core part of the draft).

* Introduce DAFs first, in order to mirror the document structure.

* Explain (function) secret sharing and give examples of aggregation functions that can be computed with (V)DAFs

* Give less detail about Prio3 and Poplar1

* Point to discussion about non-collusion in {{overview}}

Also, in security considerations, discuss dealing with non-static corruptions and how to pick the number of aggregators. This addresses an idea for the intro, but resolving it seems to require a proper security consideration.

### CODE REVIEW

File: draft-irtf-cfrg-vdaf.md [hunk 1]
```diff
@@ -5413,6 +5455,16 @@ instead, but `PROOFS` MUST be set to at least `3`. Breaking robustness for
...
 
+## Choosing the Number of Aggregators {#num-aggregators}
+
+Two Aggregators are required for privacy in our threat model, but some (V)DAFs,
+including Prio3 ({{prio3}}), allow for any number of Aggregators, only one of
+which needs to be trusted in order for the computation to be private. To hedge
+against corruptions that happen during the course of the attack, deployments
+may consider involving more than two Aggregators as described for example in
+{{star-topo}}. Note however that not all schemes, including Poplar1, are
+compatible with this mode of operation.
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:5466]: I found "including" to be confusing here, so I rephrased it.
```suggestion
{{star-topo}}. Note however that some schemes are not compatible with this mode of operation,
such as Poplar1.
```

File: draft-irtf-cfrg-vdaf.md [hunk 2]
```diff
@@ -305,65 +377,34 @@ This document also specifies two concrete VDAF schemes, each based on a protocol
...
-> OPEN ISSUE Decide if we should give one or two example DAFs. There are natural
-> variants of Prio3 and Poplar1 that might be worth describing.
-
-The remainder of this document is organized as follows: {{overview}} gives a
-brief overview of DAFs and VDAFs; {{daf}} defines the syntax for DAFs; {{vdaf}}
+* The Poplar protocol {{BBCGGI21}} solves the heavy-hitters problem in a
+  privacy-preserving manner. Here each client holds a bit-string, and the goal
+  of the aggregation servers is to compute the set of strings that occur at
+  least `t` times form some threshold `t`. The core primitive in their protocol
+  is a secret sharing of a point function {{GI14}} (`g` in the notation above)
```
[the remark this answers is not in the document]
Comment (cjpatton) [reply draft-irtf-cfrg-vdaf.md:392]: The operative word is "allows", as in "they evaluate point function shares on each of their input shares and add up the output shares to get share of the count". Is this is clear enough or do we need to fix it?

File: draft-irtf-cfrg-vdaf.md [hunk 3]
```diff
@@ -5402,7 +5444,7 @@ flexible circuit design (see {{flp-bbcggi19-overview-extensions}}), but opens up
 Prio3 to precomputation attacks, which the larger field mitigates. (See
 {{DPRS23}}, Theorem 1.) Note that privacy is not susceptible to such attacks.
 
-Another way to mitigate this issue (or improve robustness in general) is to
+Another way to mitigate this issue (or improve robustness in general) is to:wq
```
[the remark this answers is not in the document]
Comment (cjpatton) [reply draft-irtf-cfrg-vdaf.md]: 😬

File: draft-irtf-cfrg-vdaf.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (divergentdave) [reply draft-irtf-cfrg-vdaf.md:392]: I think it's fine as-is

**Review by divergentdave — APPROVED** [review]

**Comment by cjpatton** [comment]
Rebased and squashed.

**Pull request merged**